test/container/Dockerfile

```
@@ -104,13 +104,16 @@ RUN userdel -r ubuntu && \
     HOME=/home/user git config --global user.name "ArgoCD Test User" && \
     HOME=/home/user git config --global user.email "noreply@example.com" && \
     HOME=/home/user git config --global --add safe.directory '*' && \
+    mkdir -p /go && \
+    mkdir -p /go/src && \
```

We can combine the 2 lines into 1.
```suggestion
    mkdir -p /go/src && \
```